main.css

```diff
@@ -0,0 +1,149 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+body {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    width: 1200px;
+    background-color: aliceblue;
+}
+.header {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    height: 100px;
+    width: 70%;
+    background: linear-gradient(10deg, #24a4ff 50%, #2c6afb 60%);
+    border-radius: 15px 15px 0 0;
+    box-shadow: 3px 3px 3px grey;
+}
+#task {
+    width: 50%;
+    padding: 10px 20px;
+    margin-right: 20px;
+    border: none;
+    border-radius: 5px;
+    background-color: ghostwhite;
+    font-size: 18px;
+}
+.add {
+    margin-left: 20px;
+    height: 40px;
+    width: 70px;
+    border: none;
+    border-radius: 5px;
+    color: black;
+    background-color: white;
+    text-transform: uppercase;
+    cursor: pointer;
+}
+.add:hover {
+    color: #fff;
+    background-color: black;
+    transition: color .8s, background-color .8s;
+}
+.container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    width: 70%;
+    height: 500px;
+    background-color: white;
+    margin-top: 10px;
+    padding-top: 50px;
+    border-radius: 0 0 10px 10px;
+    border: 2px solid cornflowerblue;
+    border-top: none;
+    box-shadow: 3px 3px 3px grey;
+}
+
+.to-do, .doing, .done {
+    width: 250px;
+    height: 400px;
+    background-color: rgb(245, 245, 245);
+    border-radius: 10px;
+    box-shadow: 3px 3px 3px grey;
+    overflow: auto;
+}
+
+.to-do-header, .done-header, .doing-header {
+    padding-left: 85px;
+    padding-top: 10px;
+    font-size: 26px;
+    font-weight: bold;
+    text-transform: uppercase;
+    background-color: rgb(5, 5, 5);
+    color: white;
+    height: 60px;
+    border-radius: 10px 10px 0 0;
+    border-bottom: thick double darkgrey;
+}
+
+.new-div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 10px;
+    font-size: 30px;
+    height: 90px;
+    margin: 10px;
+    border-radius: 15px;
+    box-shadow: 3px 3px 3px 3px black;
+}
+
+.new-div-buttons {
+    width: 200px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 5px;
+}
+
+.arrow, .delete-button, .left-arrow, .delete-button-done, .left-arrow-done, .delete-button-doing {
```

Comment:
This is unnecessarily long. I suggest changing it to something like that:
```
.button {
  height: 25px;
  width: 25px;
  font-size: 20px;
  background-color: rgb(245, 245, 245);
  border: none;
  border-radius: 5px;
}
```
and using the `.button` class in your JavaScript and HTML.